api/admin.py

```diff
@@ -6,7 +6,9 @@
                      Item,
                      AssetMake,
                      ItemModelNumber,
-                     SecurityUser)
+                     SecurityUser,
+                     CheckinCheckout
```

Comment:
lets use `AssetLog` instead of `CheckinCheckout`

Reply:
On it